float64 begin_lat
float64 begin_long
float64 end_lat
float64 end_long
float64 leg_course
float64 xte_min
float64 xte_max
float64 currentlyAutonomous
